pom.xml

```diff
@@ -334,6 +305,56 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>java9_and_up</id>
```

Comment:
What's the best way to try this out? And out of curiosity, will the usage be different for Java 8&below vs Java 9 &up?

Reply:
The reason I set Java 9 as the cutoff is that `--add-opens` option was added in Java 9. So if you pass `-add-opens` to `java` when running Java 8, you'll get an error saying that this option is unrecognized.

You can use SDKman to switch java distributions -- I've been testing with `java 8.0.322-tem` and `17.0.1-oracle`. 

For this codebase, use `mvn clean verify` to reproduce the issue -- comment out the customizations to see the problem.

For the example in https://github.com/elefeint/examples/blob/master/appengine17-helloworld/ , you can use `mvn clean package appengine:run` -- it's a lot less indirection than launching a test with failsafe that then runs devserver.